packages/design-system/src/components/DateField/DateInput.jsx

```diff
@@ -78,6 +78,7 @@ export class DateInput extends React.PureComponent {
         }}
         autoComplete={this.props.autoComplete && `bday-${type}`}
         aria-describedby={this.props.labelId}
+        aria-invalid={this.props[`${type}Invalid`]}
```

Comment:
Because of this change, `DateField` components will use `aria-invalid` in 2 different ways. The attribute will be on the `fieldset` that contains the 3 fields, and additionally on any field that has an error state. @davidakennedy 

So the resulting markup will be something like this:
```
<fieldset aria-invalid=true>
   <legend>...</legend>
   <div>
      <input/>
      <input/>
      <input aria-invalid=true/>
   </div>
</fieldset>
```

Let me know what you think of this, if its overkill or not. We could scale it back to only using the `aria-invalid` on the `legend` if thats less confusing. We could also only use `aria-invalid` on the inputs for the `DateField` component, but right now the way it work is that `aria-invalid` is automatically added to errored components using `fieldset` and `legend`, so that would be a bit tricker to setup.

Reply:
I think ideally having it on the input only would be good but given the context I understand why it's at the fieldset level.

Reply:
> We could scale it back to only using the aria-invalid on the legend if thats less confusing

Oops I had a typo, I meant to say we could scale the `aria-invalid` back to only on the `fieldset`, the `legend` doesnt have anything.

Reply:
@davidakennedy 
Just to clarify, you are ok with having 2 instances of `aria-invalid` on the DateField component, 1 instance is on the `fieldset` and the other is on errored `input` fields